openstef/model/serializer.py

```diff
@@ -92,6 +92,11 @@ def _log_model_with_mlflow(
         mlflow.set_tag("feature_modules", model_specs.feature_modules)
         mlflow.log_metrics(report.metrics)
         model_specs.hyper_params.update(model.get_params())
+        # TODO: Remove this hardcoded hyper params fix with loop after fix by mlflow
+        # https://github.com/mlflow/mlflow/issues/6384
+        for key, value in model_specs.hyper_params.items():
+            if value == "":
+                model_specs.hyper_params[key] = " "
```

Comment:
Should this temporary fix also be made when loading in the hyperparams from a stored model?